lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -50,6 +50,9 @@ def compute(self, student, course_structure, scores_client, submissions_scores):
         ):
             self._compute_block_score(student, descendant_key, course_structure, scores_client, submissions_scores)
 
+        # self.scores may hold outdated data, force it to refresh on next access
+        lazy.invalidate(self, 'scores')
```

Comment:
Why the top? I was thinking that there could be errors in `_compute_block_score`, and we should wait to invalidate until that operation has successfully completed